src/Domain/LeanCode.DomainModels/Model/TypedIdConverterAttribute.cs

```diff
@@ -49,6 +49,25 @@ public override void Write(Utf8JsonWriter writer, Id<T> value, JsonSerializerOpt
         {
             writer.WriteStringValue(value.Value);
         }
+
+        public override Id<T> ReadAsPropertyName(
+            ref Utf8JsonReader reader,
+            Type typeToConvert,
+            JsonSerializerOptions options
+        )
+        {
+            if (reader.GetString() is string s && Guid.TryParse(s, out var id))
```

Comment:
Why not [GetGuid](https://learn.microsoft.com/en-us/dotnet/api/system.text.json.utf8jsonreader.getguid?view=net-7.0)?